File: Python/Easy/even_or_odd.py
# https://edabit.com/challenge/suhHcPgaKdb9YCrve
def even_or_odd(s):
    o,e=[],[]
    s=[int(i) for i in s]
    for i in s:
        if i%2:
            o.append(i)
        else:
            e.append(i)
    if sum(o)>sum(e):
        return 'Odd is greater than Even'
    elif sum(o)<sum(e):
        return 'Even is greater than Odd'
    else:
        return 'Even and Odd are the same'


print(even_or_odd('12345'), 'Odd is greater than Even')

File: javascript/Easy/evenOrOdd.js
// https://www.codewars.com/kata/57f7b8271e3d9283300000b4/train/javascript

function evenOrOdd(str) {
  let [even, odd] = [0, 0];
  for (let i = 0; i < str.length; i++) {
    const num = str[i];
    if (/[02468]/.test(str[i])) {
      even += +num;
    } else {
      odd += +num;
    }
  }
  return even > odd
    ? "Even is greater than Odd"
    : odd > even
    ? "Odd is greater than Even"
    : "Even and Odd are the same";
}
